app/controllers/api/v1/mixins/index_mixin.rb

```diff
@@ -22,6 +22,14 @@ def collection(base_query)
         end
 
         def rbac_scope(relation)
+          if ManageIQ::API::Common::RBAC::Roles.assigned_role?('Catalog Administrator')
```

Comment:
The fact that this is method call exists twice and is passing a string makes me think:
1. It should be a method with a simple, more descriptive name
2. The string role name should be a constant.

Can this live in the `rbac_mixin` and define an `ADMINISTRATOR_ROLE_NAME = 'Catalog Administrator'` and a method like 

```ruby
ADMINISTRATOR_ROLE_NAME = 'Catalog Administrator'.freeze
...

def catalog_administrator?
  ManageIQ::API::Common::RBAC::Roles.assigned_role?(ADMINISTRATOR_ROLE_NAME)
end
```

Reply:
Updated to use the constant in the specs as well as in the code